ezbake/config/conf.d/auth.conf

```diff
@@ -75,6 +75,21 @@ authorization: {
             sort-order: 500
             name: "puppetlabs cert statuses"
         },
+        {
+            # Allow the CA CLI to access the certificate clean endpoint
+            match-request: {
+                path: "/puppet-ca/v1/clean"
+                type: path
+                method: [put]
```

Comment:
(nit) Based on other entries in here, it looks like single methods don't need to be wrapped in brackets.